README.md

```diff
@@ -64,73 +63,79 @@ Version 10.17.0
+Best practices (though not required) suggest using Node Version Manger (nvm) to aid in managing which Node runtime to use during development. Through nvm, you can quickly migrate between different node versions for base development and/or testing purposes. Depending on your OS (\*Nix vs Windows) installation of a nvm varies.
 
 Relevant packages can be found here:
 
 - [NVM for Windows](https://github.com/coreybutler/nvm-windows)
-
 - [NVM for Mac](https://github.com/nvm-sh/nvm)
 
 ### Installation
 
-To install packages run
+To install packages, in the repo directory run:
 
+```bash
+$ npm install
 ```
-$ npm install or npm i
+
+or
+
+```bash
+$ npm i
 ```
 
 #### Note: Pre and Post Installation
 
-When installing you'll notice a pre and post install script run. These scripts are for **_QA_**/**_Production_**/**_Staging_** environments and can be ignored. However, if the scripts fail it may indicate you're running a different version of Node's runtime environment. Reference the projects [Node Runtime](#node-runtime)
+When installing you'll notice a pre and post install script run. These scripts are for **_QA_**/**_Production_**/**_Staging_** environments and can be ignored. However, if the scripts fail it may indicate you're running a different version of Node's runtime environment. Reference the project's current [Node Runtime](#node-runtime).
 
 ### Configurations
 
-There are a few consideration to be aware of when spinning up a local development environment. And, while you will be able to view a local version on your machine with just setting up the environment variables, the application may soon break if the host isn't configured properly.
+There are a few consideration to be aware of when spinning up a local development environment. While you will be able to view a local version on your machine with just setting up the environment variables, the application may soon break if the host isn't configured properly.
 
 #### Node
 
-Ensure you are running the proper [node version](#node-runtime).
-If missconfigured, there will be issues with webpack building the project due to how **_`sass`_** is configured
+Ensure you are running the proper [node version](#node-runtime). If misconfigured, there will be issues with webpack building the project due to how **_`sass`_** is configured.
 
-#### Environment
+#### Environment Variables
 
-See `.env-sample` for supported environmental variables. Rename `.env-sample` to `.env` and replace placeholder values with your own - or obtain a prefilled version from a coworker.
+See `.env-sample` for supported environmental variables. Copy and rename `.env-sample` as `.env` and replace placeholder values with your own - or obtain a prefilled version from a coworker.
```

Comment:
We should probably just say `Copy .env-sample to .env`, we actually _don't_ want the original file to be renamed